app/js/instances/createContainerController.js

```diff
@@ -0,0 +1,59 @@
+var _ = require('lodash');
+
+function createContainerController($scope, $routeParams, $location, dockerService, instanceModel) {
+    'use strict';
+    $scope.host = $routeParams.host;
+
+    dockerService.images.list($scope.host, null);
+
+    $scope.$listenTo(instanceModel, function () {
+      $scope.images = _.map(instanceModel.getImages(), function(image) {
+
+        image.Tags = _.filter(image.RepoTags, function(repoTag) {
+          return repoTag !== '<none>:<none>';
+        }).join(', ');
+
+        return image;
+      });
+
+      $scope.images = _.filter($scope.images, function(image) {
+        return image.Tags !== '';
+      });
+
+      $scope.selectedImage = $scope.images[0];
+    });
+
+    $scope.enviromentVariables = [];
+    $scope.enviromentInput = "";
+    $scope.cmdInput = "";
+
+    $scope.addEnviromentVar = function(event) {
+      if ((event.which !== 13 && event.type === "keypress") || $scope.enviromentInput === "") {
```

Comment:
We can possibly remove this UI paradigm if it's not optimal for code.  Something like comma delimited input could work, but it's not as fancy.  For example, ```Foo=Bar, Left=Shark"...